c2corg_ui/static/js/documentediting.js

```diff
@@ -536,31 +577,52 @@ app.DocumentEditingController.prototype.updateMaxSteps = function(waypointType)
  * @param {Object} outing
  * @private
  */
-
 app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
-  // creating a new outing -> init locales
+  if (this.submit_) {
+    if (outing['locales'][0]['conditions_levels'][0]['level_place'].length > 0) {
```

Comment:
I don't think so, as I said this will surely be part of the PR where we will factorize the code into document-specific controllers, and there are already issues created by you about this.